.github/project-community-pr-assigner.yml

```diff
@@ -85,11 +88,15 @@
 "plugins/woocommerce/src/Internal/Admin/**/*":
   - team: mothra
   - team: ghidorah
-  
+
 "plugins/woocommerce-admin/**/*":
   - team: mothra
   - team: ghidorah
 
+"plugins/woocommerce-blocks/**/*":
+  - team: rubik
+  - team: woo-fse
```

Comment:
yaml looks valid, but we don't have any PRs or issues with this `team: woo-fse` label. Is that being monitored by anyone?

Reply:
Those are Github teams not labels: `woocommerce/rubik` and `woocommerce/woo-fse`.